### PR TITLE
Harassment allegations

### PR DESCRIPTION
**Pull request by jokudasai**
Update hp-action-harassment (page 2 of 2)

### CODE REVIEW

File: frontend/lib/pages/hp-action-harassment.tsx [hunk 1]
```diff
@@ -82,23 +81,23 @@ export const HarassmentAllegations2 = stepBuilder.createStep(props => ({
...
     <CheckboxFormField {...ctx.fieldPropsFor('allegRemovedPossessions')}>
-      removed tenant possessions from the unit, or removed the unit front door or made the lock to the unit not work, or changed the lock on the unit door without giving a key to the new lock to the tenant/petitioner
+      removed your belongings, the front door, or locked you out of your apartment
     </CheckboxFormField>
     <CheckboxFormField {...ctx.fieldPropsFor('allegInducedLeaving')}>
-      offered money or valuables to tenant, or their relatives, to induce tenant to leave, or to surrender or waive their rights, without written disclosure of the tenant’s rights and without written permission to make an offer from court or the tenant; or, while: threatening, intimidating or using obscene language; frequently harassing or communicating abusively; communicating at tenant’s place of employment without prior written consent; or  knowingly falsifying or misrepresenting information to tenant
+      made buyout offers while threatening you - OR - Cursed or used profane language to intimidate you - OR - Knowingly lied to you - OR - Contacted your job without your permission
```
Comment (toolness) [inline frontend/lib/pages/hp-action-harassment.tsx]: Should the statements after the "- OR -"'s be un-capitalized?  (Same question goes for the checkboxes elsewhere that have "- OR -" in them)

File: frontend/lib/pages/hp-action-harassment.tsx [hunk 2]
```diff
@@ -52,25 +52,24 @@ export const HarassmentAllegations1 = stepBuilder.createStep(props => ({
   renderIntro: renderAllegationsIntro,
   renderForm: ctx => <AllegationsFieldset>
     <CheckboxFormField {...ctx.fieldPropsFor('allegForce')}>
-      Used force or threatened to use force
+      used force or threatened to use force
     </CheckboxFormField>
     <CheckboxFormField {...ctx.fieldPropsFor('allegMisleadingInfo')}>
-      knowingly provided false or misleading information on the current occupancy, or rent stabilization status of a unit on any application or construction document for a permit for work to be performed in said building
+      lied about occupancy or rent status of your apartment
```
Comment (toolness) [inline frontend/lib/pages/hp-action-harassment.tsx]: Should this actually say "lied about the occupancy or rent status of your apartment"?  (I added the word "the")

File: frontend/lib/pages/hp-action-harassment.tsx (hunk 1 above)
Comment (jokudasai) [reply frontend/lib/pages/hp-action-harassment.tsx]: I'd like for the "or"s to be bold and lowercase but I didn't know how to do that. thanks for your help

File: frontend/lib/pages/hp-action-harassment.tsx (hunk 2 above)
Comment (jokudasai) [reply frontend/lib/pages/hp-action-harassment.tsx]: yes, that's a good catch!

**Pull request merged**